coops-ui/src/views/AnnualReport.vue

```diff
@@ -159,6 +159,8 @@
                   :certifiedBy.sync="certifiedBy"
                   :currentDate="currentDate"
                   @valid="certifyFormValid=$event"
+                  :entityDisplay="this.configObject.displayName"
+                  :message="this.configObject.flows.find(x => x.feeCode === 'OTANN').certifyText"
```

Comment:
I think the method to extract the data should be in a common module (similar to how we use store), so all components that need certifyText could call something like `getCertifyText(x.feeCode)`.

It also feels like this functionality is global/common/useful enough that any component in the app should be able to use it easily, possibly with a mixin? Or attached to Vue itself?